geopandas/geoseries.py

```diff
@@ -382,7 +382,7 @@ def _wrapped_pandas_method(self, mtd, *args, **kwargs):
     def __getitem__(self, key):
         return self._wrapped_pandas_method("__getitem__", key)
 
-    @inherit_doc(pd.Series)
+    # @inherit_doc(pd.Series) - doctest failure
```

Comment:
I was not able to force pytest to skip inherited docstring (which fails), so I am temporarily disabling this. Will push fix to pandas.